packages/flutter/lib/src/services/system_channels.dart

```diff
@@ -277,6 +277,33 @@ abstract final class SystemChannels {
       JSONMethodCodec(),
   );
 
+  /// A [MethodChannel] for handling Android Scribe stylus handwriting input.
+  ///
+  /// Android's Scribe feature allows writing directly on top of a text input
+  /// using a stylus.
+  ///
+  /// The following methods are defined for this channel:
+  ///
+  ///  * `Scribe.startStylusHandwriting`: Indicates that stylus input has been
```

Comment:
nit: [] instead of ``?

Reply:
Not sure why I didn't do that in the first place...

Reply:
Actually it should be tick marks, because this refers to the platform channel method, not the method on the Scribe class.